docs/en_US/Tutorial/HowToUseSharedStorage.rst

```diff
@@ -0,0 +1,49 @@
+**How to Use Shared Storage**
+=============================
+
+If you want to use your own storage during using NNI, shared storage can satisfy you.
+Instead of using training service native storage, shared storage can bring you more convenience.
+All the information generated by the experiment will be stored under ``/nni`` folder in your shared storage.
+All the output produced by the trial will be located under ``/nni/{EXPERIMENT_ID}/trials/{TRIAL_ID}/nnioutput`` folder in your shared storage.
+This saves you from finding for experiment-related information in various places.
+Remember that your trial working directory is ``/nni/{EXPERIMENT_ID}/trials/{TRIAL_ID}``, so if you upload your data in this shared storage, you can open it like a local file in your trial code without downloading it.
+And we will develop more practical features in the future based on shared storage.
+
+.. note::
+    Shared storage is currently in the experimental stage. We suggest use AzureBlob under Ubuntu/CentOS/RHEL, and NFS under Ubuntu/CentOS/RHEL/Fedora/Debian for remote.
+    And make sure your local machine can mount NFS or fuse AzureBlob and has ``sudo`` permission on your remote runtime. We only support shared storage under training service with reuse mode for now.
+
+Example
+-------
+If you want to use AzureBlob, add below to your config. Full config file see :githublink:`mnist-sharedstorage/config_azureblob.yml <examples/trials/mnist-sharedstorage/config_azureblob.yml>`.
+
+.. code-block:: yaml
+
+    sharedStorage:
+        storageType: AzureBlob
+        localMountPoint: ${your/local/mount/point}
+        remoteMountPoint: ${your/remote/mount/point}
+        storageAccountName: ${replace_to_your_storageAccountName}
+        storageAccountKey: ${replace_to_your_storageAccountKey}
+        # If you did not set storageAccountKey, you need use `az login` with Azure CLI at first and set resourceGroupName.
+        # resourceGroupName: ${replace_to_your_resourceGroupName}
+        containerName: ${replace_to_your_containerName}
```

Comment:
What does containerName mean? Is it a contain id?

Reply:
The name of the container in blob service in your storage account, like `nni`.
![image](https://user-images.githubusercontent.com/33053116/109102179-3c2d4700-7763-11eb-8595-d76ee818792e.png)